src/deployed.json

```
@@ -2127,6 +2127,9 @@
                 "0xE6c2C240E1228980F322D76ed4C1F63078191BB5",
                 "0xeEEE2a622330E6d2036691e983DEe87330588603",
                 "0xF0FAC7104aAC544e4a7CE1A55ADF2B5a25c65bD1"
+            ],
+            "fakeToken": [
+                "0xb81D3cB2708530ea990a287142b82D058725C092"
```

```suggestion
               "0xa1d0E215a23d7030842FC67cE582a6aFa3CCaB83",
               "0xb81D3cB2708530ea990a287142b82D058725C092"
```

This token seems to be a fake YFI, makes sense to add it as well.

---

I'd be happy to make this change, but it seems like this PR is now duplicated by #227 . You can feel free to close. Thanks!